hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -1547,17 +1547,16 @@ public BootstrapStateHandler.Lock getBootstrapStateLock() {
     return lock;
   }
 
-  public String pngPrintMutableGraph(String fileName, GraphType graphType)
+  public void pngPrintMutableGraph(String filePath, GraphType graphType)
       throws IOException {
-    Objects.requireNonNull(fileName, "Image file name is required.");
+    Objects.requireNonNull(filePath, "Image file path is required.");
     Objects.requireNonNull(graphType, "Graph type is required.");
 
     PrintableGraph graph;
     synchronized (this) {
-      graph = new PrintableGraph(forwardCompactionDAG, graphType);
+      graph = new PrintableGraph(backwardCompactionDAG, graphType);
```

Comment:
Why the switch? Or should we just print both?

Reply:
backwardCompactionDAG will give the same result as in https://user-images.githubusercontent.com/6820020/211372896-c4f6eada-3fb0-4838-bcc6-08c47606a43f.png

Attaching the images to differentiate.

1. backwardCompactionDAG:
![backwardCompactionDAG](https://github.com/apache/ozone/assets/6820020/6dd1f8ce-c52d-444c-9b99-03f087478892)

2. forwardCompactionDAG:
![forwardCompactionDAG](https://github.com/apache/ozone/assets/6820020/20c1db56-e8e4-43bd-9cb3-69579bf64e85)